src/hooks/useCourtStats.js

```diff
@@ -0,0 +1,48 @@
+import { useMemo } from 'react'
+import { useTotalActiveBalance, useTotalANTStaked } from './useCourtContracts'
+import { useActiveJurorsNumber } from '../hooks/query-hooks'
+import { getKnownToken } from '../utils/known-tokens'
+import { bigNum } from '../lib/math-utils'
+import IconANJ from '../assets/IconANJ.svg'
+import IconANT from '../assets/IconANT.svg'
+
+const COURT_STATS = [
+  {
+    label: 'Total Active ANJ',
+    token: { ...getKnownToken('ANJ'), icon: IconANJ },
+  },
+  {
+    label: 'Total Staked ANT',
+    token: { ...getKnownToken('ANT'), icon: IconANT },
+  },
+  { label: 'Total Active Jurors' },
+]
+
+function useCourtStats() {
```

Comment:
It would be nice to add a bit of JSDoc to hooks like this, especially for hooks where we're returning tuples with the loading state in the second index.